package.json

```diff
@@ -121,7 +121,6 @@
     "babel-preset-preact": "^2.0.0",
     "chart.js": "^2.9.3",
     "clipboard-polyfill": "^2.8.6",
-    "codemirror": "^5.52.2",
```

Comment:
the PR is missing the updated `yarn.lock`

Reply:
On it !!